packages/affiliates/test/mocha-local/liquidity-mining/CalculateRollBalancerLPRewards.js

```diff
@@ -367,7 +376,7 @@ contract("CalculateBalancerLPProviders.js", function (accounts) {
 
       // Capture the starting block number.
       const startingBlockNumber = await web3.eth.getBlockNumber();
-      const startingBlockTimestamp = await web3.eth.getBlock(startingBlockNumber).timestamp;
+      const startingBlockTimestamp = (await web3.eth.getBlock(startingBlockNumber)).timestamp;
```

Comment:
hhhmm. was this an error before?

Reply:
Yes! It was a bug that just happened to be working I think. It was undefined, which was implicitly being converted to the number 0. Mining blocks near 0 (before existing blocks) is allowed in ganache, but hardhat disallows this, which is why I think this worked previously, but no longer does.

Reply:
^This is just a guess as to what was going on, but I'm confident that the previous syntax was not intended.

Reply:
ye, previously was a bug before for sure. your analysis makes sense methinks

Reply:
wow nice catch